static/app/views/performance/http/httpLandingPage.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {Fragment} from 'react';
+import styled from '@emotion/styled';
+import pickBy from 'lodash/pickBy';
```

Comment:
Can we write this without lodash? There's an ongoing frontend tsc issue to remove lodash.

Reply:
Using `reduce` for keyBy is a loss of readability I'm not comfortable with. Do we have  an equivalent to `keyBy` in our own code? If not, this is better as-is IMO, or I can add a `keyBy` helper in another PR

Reply:
`keyBy` helper in another PR sounds good to me